capstone/src/main/webapp/dashboard.html

```diff
@@ -3,19 +3,57 @@
   <head>
     <meta charset="UTF-8">
     <title>Dashboard</title>
+    <link href="https://fonts.googleapis.com/css2?family=Vollkorn:wght@400;600;800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
   </head>
   <body>
 
-    <div id="content">
-      <h1>Dashboard</h1>
-      <input type="text" id="subject" placeholder="Subject"> 
-      <button onclick="createClassroom(document.getElementById('subject').value)">Create Classroom</button> 
+    <nav class="navbar navbar-default">
+        <ul class="nav navbar-nav navbar-left">
+            <li><a class="navbar-brand" href="index.html">Rooms</a></li>
+        </ul>
+        <div class="text-inline">
+            <ul class="navbar-nav navbar-expand-lg">
+                <li class="nav-item margin-right">
+                    <button type="button" class="btn btn-default" data-toggle="modal" data-target="#exampleModalCenter">
+                        <img src="/assets/plus.svg">  Create
+                    </button>
+                </li>  
+                <li class="nav-item"><div id="signout"></div></li>
+            </ul>
+        </div>
+    </nav>
+ 
+    <div class="modal fade" id="exampleModalCenter" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true">
```

Comment:
This could be named something better than "exampleX"

Reply:
This was changed in a later PR to the specific kind of lesson but for now I will just call it ModalCenterLesson